.golangci.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
(nitpick) this probably is better suited for the next commit, i.e. this one cleans up the existing code to be "lint" ready, the next "adds the linting" but it seems this file fits better into the part(?)

Reply:
please check